code/pld/r2/r2020_cpu_glue/ic8_cpu_glue.pld

```diff
@@ -0,0 +1,33 @@
+GAL16V8    ; For ATF16V8BQL
+Glue       ; CPU glue logic for Classic 68020 edition
+
+Clock DS    SIZ0    SIZ1    A0    AS   NC    NC    NC   GND
+/OE   NC    LDS     UDS     NC    Q3   Q2    Q1    Q0   VCC
+
+Q0.R = /Q0
+
+
+Q1.R = /Q1 *  Q0
+     +  Q1 * /Q0
+
+
+Q2.R =  Q2 * /Q1
+     +  Q2 * /Q0
+     + /Q2 *  Q1 *  Q0
+
+
+Q3.R =  Q3 * /Q2
```

Comment:
One thing that's probably very minor, is the E clock being low for 8 clock cycles, and high for 8, instead of low for 6, high for 4. I might take a crack at the PLD equations myself later to avoid adding too much work over such a small thing.

Reply:
Follow-up to this added as separate comment.

Reply:
Changes made (full reply and proof-of-life in other comment), thanks @0xTJ 👍 :)